cumulus_library/actions/builder.py

```diff
@@ -203,13 +243,23 @@ def build_matching_files(
     *,
     builder: str | None,
     db_parser: databases.DatabaseParser = None,
+    prepare: bool = False,
+    data_path: pathlib.Path,
```

Comment:
Same comment here about default args - This feature feels like a thing that a future dev (including us) might totally forget exists when adding code that calls these methods. If everyone calling this is passing `prepare=xxx` right now (though I did not check if that were true here, to be fair), I'm guessing we'll want all callers in future to pass it in too.